gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,65 @@
+//tag::intro1[]
+
+Flex Gateway includes the following CLI command to test gateway health. You can run this command manually on any Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either an exit code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
+
+//end::intro1[]
+
+//tag::intro2[]
+You can also configure this command to run automatically. By default, Flex Gateways deployed on Kubernetes have a liveness probe configured. The liveness probe periodically runs the liveness check command and automatically restarts the Flex Gateway pod after a specified number of failed liveness checks. 
+
+Flex Gateways deployed on Docker or Linux, Flex Gateway does not offer a default liveness probe. However, you can still use the the liveness check command. For both deployments, you can use liveness check in scripts that check liveness and restart containers. 
```

Comment:
IMHO:
- I would avoid saying that "Flex Gateway does not offer a default liveness probe." because I'm afraid that users might think that they don't have a way to configure a liveness probe. Maybe something like "Flex Gateway does not configure a default liveness probe."
- I would avoid using the word "script" here, since liveness probes can be configured through other ways too

```suggestion
Flex Gateways deployed on Docker or Linux do not configure a default liveness probe. However, you can still use the the liveness check command. For both deployments, you can use liveness check to check liveness and restart Flex Gateway. 
```